translations/en-us.yml

```diff
@@ -896,6 +896,30 @@ institutions:
             first: First
             prev: Prev
             next: Next
+            request-project-message-modal:
+              tabs:
+                request_access: 'Request Access'
+                send_message: 'Send Message'
+              open_aira_label: 'Open Project Request Modal Button'
```

Comment:
Nit: I think it's `aria` instead of `aira`, but you're using the correct keys it seems, so nothing should break because of this